stacks/vitess/README.md

```diff
@@ -0,0 +1,78 @@
+# Getting Started with the Vitess Operator
+
+After you have deployed this stack you will have the Vitess operator and an
+example Database cluster. 
+
+## Base Requirements
+The example database requires four persistent volume claims ( 3 for the
+lockserver, and 1 for the database ). It also requires three nodes to function
+effectively. The operator will make its best effort to spread all components
+across nodes to ensure availability. 
+
+## Accessing the Cluster
+To log into this cluster you can use the following commands
```

Comment:
Suggestion: "To log into the example database cluster, use the following commands:"